python-sdk/src/astro/sql/__init__.py

```diff
@@ -4,14 +4,11 @@
 
 from astro.sql.operators.append import AppendOperator, append
 from astro.sql.operators.cleanup import CleanupOperator, cleanup
-from astro.sql.operators.data_validations.ColumnCheckOperator import (  # skipcq: PY-W2000
+from astro.sql.operators.data_validations.column_check import (  # skipcq: PY-W2000
     ColumnCheckOperator,
     column_check,
 )
-from astro.sql.operators.data_validations.SQLCheckOperator import (  # skipcq: PY-W2000
-    SQLCheckOperator,
-    sql_check,
-)
+from astro.sql.operators.data_validations.sql_check import SQLCheckOperator, sql_check  # skipcq: PY-W2000
```

Comment:
```suggestion
from astro.sql.operators.data_validations.sql_check import SQLCheckOperator, check_sql  # skipcq: PY-W2000
```

Reply:
@pankajkoti  I think it should be `sql_check` since the operator is checking based on the SQL provided as input and `check_sql` gives it a different intent of checking SQL. WDYT?

Reply:
It is ambiguous to me. What do we intend `sql_check` to mean? `sql_check = check_sql` to me. 
And in general method names should be verbs https://www.codementor.io/@tanakamutakwa/writing-clean-code-naming-1msxn728ht#methodsfunctions-should-have-verb-or-verb-phrase-names

We're intending to check something, so it should begin with check and then append to it what we mean to check, no?